.github/workflows/ci.yml

```diff
@@ -6,52 +6,21 @@ on:
     branches:
       - main
 
-env:
-  RUST_BACKTRACE: 1
-  CARGO_TERM_COLOR: always
-
 jobs:
```

Comment:
Perhaps add a test that the TypeScript source code matches the JavaScript in the `dist` directory? Otherwise we might get into a bad situation where we think something is implemented but we've forgotten to recompile and it isn't actually running.

Reply:
Great idea! I've already made this mistake once or twice...

Reply:
Aaaand done! It would be great if you could take a look :)

Reply:
Neat! I trust you've already tested that this "fails correctly" and catches the mistake we intended it to catch?

Reply:
Sure! I'm developing things like these on my private fork to make this pull request clean. This is a draft PR: https://github.com/mgr0dzicki/cargo-semver-checks-action/pull/8, and especially the "correctly failing" workflow: https://github.com/mgr0dzicki/cargo-semver-checks-action/actions/runs/4232011758/jobs/7351233661.